internal/version/version.go

```diff
@@ -130,26 +127,9 @@ func (v Info) Proto() (*pb.NodeVersion, error) {
 	}, nil
 }
 
-// containsVersion compares the allowed version array against the passed version
-func containsVersion(all []SemVer, x SemVer) bool {
-	for _, n := range all {
-		if x == n {
-			return true
-		}
-	}
-	return false
-}
-
-// StrToSemVerList converts a list of versions to a list of SemVer
-func StrToSemVerList(serviceVersions []string) (versions []SemVer, err error) {
-	for _, subversion := range serviceVersions {
-		sVer, err := NewSemVer(subversion)
-		if err != nil {
-			return nil, err
-		}
-		versions = append(versions, *sVer)
-	}
-	return versions, err
+// matchesVersion compares and checks if the passed version is greater/equal than the minimum required version
+func matchesVersion(test SemVer, target SemVer) bool {
```

Comment:
matchesVersion doesn't seem to be the right name this. Maybe `isNewerThan` or similar?

Reply:
Adressed :)